api/src/main/java/org/semanticweb/owlapi/io/AbstractOWLParser.java

```diff
@@ -95,6 +95,9 @@ protected InputStream getInputStream(@Nonnull IRI documentIRI,
         String originalProtocol = originalURL.getProtocol();
         URLConnection conn = originalURL.openConnection();
         conn.addRequestProperty("Accept", actualAcceptHeaders);
+		if (config.isAcceptingAuthorization()) {
```

Comment:
Tabs are not used by this codebase, please replace each tab with 4 spaces.

Reply:
I have updated source and replaced each tab with 4 spaces.